src/model/enhance-interactive-content-elements.ts

```diff
@@ -0,0 +1,87 @@
+import { stripHTML } from '@root/src/model/strip-html';
+
+const scriptUrls = [
+	'https://interactive.guim.co.uk/page-enhancers/nav/boot.js',
+	'https://uploads.guim.co.uk/2019/03/20/boot.js',
+	'https://uploads.guim.co.uk/2019/12/11/boot.js',
+];
+
+const isInteractiveContentBlockElement = (element: CAPIElement): boolean =>
+	element._type ===
+		'model.dotcomrendering.pageElements.InteractiveBlockElement' &&
+	scriptUrls.indexOf(element?.scriptUrl || '') !== -1;
```

Comment:
I'm not sure we have any interactive elements without script URLs but this line is a little risky because it'd identify any interactive block without a script URL as a contents block element. Maybe safer to do `element.scriptUrl && scriptUrls.indexOf(...`

Reply:
`scriptUrls.indexOf('')` would still return -1, but I think your method is still a cleaner solution so I'll get that changed

Reply:
I executed that in my console and it gave me zero - would be interested to know if you've seen different!

Reply:
![image](https://user-images.githubusercontent.com/9575458/123280207-813e6b80-d500-11eb-9e29-173ddb7fd555.png)
Very odd! Perhaps an inconsistency between versions of node / browsers?

Reply:
Sorry, my bad, I was reading it as `scriptUrl.indexOf('')` as in `String.indexOf()`!